TESTING.md

```diff
@@ -328,3 +321,73 @@ imported packages: **o1, o1.llama, o1.randomtext**.
 **8.4** Double click **SwingExamples** in **Modules** list.
 
 **8.5** ASSERTION: **SwingExamples** shows up as a module in the project tree.
+
+### 9 Authentication with A+ LMS
+
+**9.1** From the main menu, choose **A+ > Set A+ Token**.
+
+![Set token](images/setToken.png)
+
+**9.2** Click on the highlighted "What's my token?" link.
+
+**9.3** ASSERTION: Browser opens your profile page in A+ LMS (login, if asked).
+
+**9.4** Copy **API Access Token** from the web page and insert it into the prompt, click "OK".
+
+**9.5** ASSERTION: The assignments tab is populated with Weekly assignments.
+
+![Assignments shown](images/assignments-shown.png)
+
+**9.6** Close the IDE.
+
+**9.7** Open the IDE with the project.
+ 
+**9.8** If [safe password storage](https://jetbrains.org/intellij/sdk/docs/basics/persisting_sensitive_data.html?search=sensitive#storage) 
+is activated in the OS.
+
+**9.9** ASSERTION: Assignments tab is still populated.
+
+### 10 Assignments
+
+#### Part I: Submitting an Assignment.
+
+**10.1** Expand **Week 1** and select **Assignment 2 (GoodStuff)**, and hit 
+**Submit A+ Assignment**.
+
+![Submit](images/submit_button.png)
+
+**10.2** ASSERTION: The "Could not find module: The A+ Courses plugin could not find the 
+module GoodStuff." warning is shown.
```

Comment:
Nit: there is a separate notification type for warnings, so this may be confusing since the notification is actually an error notification

Reply:
actually a thing that would be nice to-re-check (added as a sub-task for https://github.com/Aalto-LeTech/intellij-plugin/issues/249)

Reply:
Yep they should be checked. "Error" is definitely correct for the missing module case IMO